docs/spelling_wordlist.txt

```diff
@@ -1,9 +1,10 @@
-acyclic
-admin
+onacyclic
```

Comment:
is this a word?

Reply:
I will make a follow-up PR to fix all of this in the renku-python repo

Reply:
oops that was indeed a weird one.